frontend/src/components/inputs/NewsLetterButton/index.js

```diff
@@ -73,8 +74,9 @@ const NewsLetterButton = ({
             >
                 This is a perfect opportunity to join our mailing list, where we
                 send very occasional (monthly) updates about our upcoming events
-                and other things happening around the Junction community. Care
-                to join?
+                and other things happening around the{' '}
+                {config.PLATFORM_OWNER_NAME}
+                community. Care to join?
```

Comment:
This will still subscribe people to the Junction newsletter without changes to the corresponding backend route :D I mean fine by me, but you would probably want that to be customisable ;)

Reply:
Oh you are right! I missed the newsletterId param 😄 I thought it was subscribing to some default Sendgrid mailing list of the tenant.

Reply:
So I checked this now with a new Sendgrid list id, and it seems that the current API used in the platform is broken. It has been 'deprecated', replaced with a New Marketing API call, that can subscribe and create recipients at the same time. I have replaced it in a new branch and will make a PR for that separately.